lib/src/utils.rs

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::fmt::Write;
+
+use anyhow::anyhow;
+use indent_write::fmt::IndentWriter;
+
+/// A temporary hack to convert a list of anyhow errors into a single
+/// `anyhow::Error`. If no errors are provided, panic (this should be handled
+/// at a higher level).
+///
+/// Eventually we should gain first-class support for representing errors as
+/// trees, but this will do for now.
+pub(crate) fn merge_anyhow_list<I>(errors: I) -> anyhow::Error
```

Comment:
This is copied from some code I wrote in Omicron. I may want to put this in a more central location in the future.